address-space-controller/src/main/java/io/enmasse/controller/PodDistruptionBudgetController.java

```diff
@@ -114,13 +114,13 @@ private void reconcileBrokerPodDisruptionBudget(AddressSpace addressSpace, Stand
                     changed = true;
                 }
 
-                Integer minAvailable = infraConfig.getSpec().getRouter().getMinAvailable();
```

Comment:
not directly related with this, but we should change the types of minAvailable and maxUnavailable to IntOrString, to get all the expressivity that allows the underliying PodDistruptionBudget, I already was prepairing a PR with that, so tell me please if I can continue and create my PR

Reply:
@famartinrh Please continue :)